.github/workflows/build.yml

```diff
@@ -24,7 +24,6 @@ run-name: '${{ inputs.bashbrewArch }}: ${{ inputs.firstTag }} (${{ inputs.buildI
 permissions:
   contents: read
   actions: write # for https://github.com/andymckay/cancel-action (see usage below)
-  id-token: write # for AWS KMS signing (see usage below)
```

Comment:
Removing this will make the "Configure AWS" and "Sign" steps fail, right? So, if this is merged as-is, then any signing runs would fail (though, admittedly, there is only one and its base images are not updated anymore).

Are there more changes to move the signing steps to a separate job in the `jobs` list (and then include this permission there)?

Reply:
Yes there are more changes coming! That change depends on the other PRs